README_dev.md

```diff
@@ -106,7 +106,7 @@ npm run test:local -- testSuiteName
           "platformClientId": "...",
           "platformClientSecret": "..."
       },
-      "database": { "driver": "lokijs", "folder": "./" },
+      "database": { "driver": "lokijs", "folder": "./", "dbName": "local" },
```

Comment:
why? we have a default for that.

Reply:
nop for me, it was not working without that config the default is only in the docker compose file I suppose

Reply:
![image](https://github.com/nmshd/connector/assets/33655937/33b6ba2b-9fbf-48ad-866f-8b65eee03871)
we have a default, we have to check elsewhere why it doesn't pulls this default for you.

Reply:
(but for now, please remove it from here)